basic/atm/ATM solve .py

```diff
@@ -0,0 +1,92 @@
+__author__ = 'Shaked Manor'
+# takes customers information from file' then
+# creating objects from class customers with the customers information
+# takes input from user, checking his ID ans password and then
+# let the customer commit actions on his account
+END_VALUE = -1
+customer_passwords = {}
+customer_access_to_class = {}
+
+
+class customers:
+    """defining the actions that a customer can do in is account,
+     checking his balance, withdrawal and deposit money and change password"""
+
+    def __init__(self):
+        self.y = 0
+        self.z = 0
+
+    def get_balance(self):
+        return self.z
```

Comment:
x, y, z, aren't such good names, you might wanna choose something more informative.